steam_wrapper/steam_wrapper.py

```diff
@@ -468,4 +468,4 @@ def main(steam_binary=STEAM_PATH):
         configure_shared_library_guard()
         enable_extensions(current_info)
         enable_discord_rpc()
-        os.execv(steam_binary, [steam_binary] + sys.argv[1:])
+        os.execv(steam_binary, [steam_binary, '-no-cef-sandbox'] + sys.argv[1:])
```

Comment:
Ah, I hadn't tried a fresh installation.

Reply:
I'm inclined to just mergethis as is and make follow-up PR unless @smcv wants to contribute fix for the other codepath as well since this PR contains a quite an understandable summary of the problem.

Reply:
I'll look at doing the other code path too, one moment.